ACKategories-iOSTests/UINavigationControllerTests.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  UINavigationControllerTests.swift
+//  ACKategories-iOSTests
+//
+//  Created by Jakub Olejník on 08.12.2020.
+//
+
+import ACKategories
+import UIKit
+import XCTest
+
+final class UINavigationControllerTests: XCTestCase {
+    func testPushCompletionWhenEmpty() {
+        let navigationController = UINavigationController()
+        let pushExpectation = expectation(description: "Push completion")
+        
+        navigationController.pushViewController(UIViewController(), animated: true) {
+            pushExpectation.fulfill()
+        }
+        
+        wait(for: [pushExpectation], timeout: 0.5)
```

Comment:
I am not sure how fast these tests are - maybe it'd make sense to introduce something like `UIApplication.shared.keyWindow?.layer.speed = 100` ?